system/loggerd/tests/loggerd_tests_common.py

```diff
@@ -105,8 +107,11 @@ def tearDown(self):
         raise
 
   def make_file_with_data(self, f_dir: str, fn: str, size_mb: float = .1, lock: bool = False,
-                          xattr: Optional[bytes] = None) -> Path:
+                          upload_xattr: Optional[bytes] = None, preserve_xattr: Optional[bytes] = None) -> Path:
```

Comment:
why don't the xattr args take bools instead?

Reply:
We have tests to make sure that setting them to `b'0'` are treated as unset. https://github.com/commaai/openpilot/pull/28206

Could make it a union and accept both for this helper?

Reply:
nah, this is fine then